OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoObjectivesLogic.cs

```diff
@@ -43,8 +52,8 @@ public GameInfoObjectivesLogic(Widget widget, World world)
 			}
 
 			var missionStatus = widget.Get<LabelWidget>("MISSION_STATUS");
-			missionStatus.GetText = () => player.WinState == WinState.Undefined ? "In progress" :
-				player.WinState == WinState.Won ? "Accomplished" : "Failed";
+			missionStatus.GetText = () => player.WinState == WinState.Undefined ? modData.Translation.GetFormattedMessage(InProgress) :
+				player.WinState == WinState.Won ? modData.Translation.GetFormattedMessage(Accomplished) : modData.Translation.GetFormattedMessage(Failed);
```

Comment:
No need to retranslate 60 times per second. Define `var accomplished = modData.Translation.GetFormattedMessage(Accomplished)` etc outside the closure for use here.